SocialFilter/sections/specific.txt

```diff
@@ -19,6 +19,7 @@ forvo.com,isyatirim.com.tr,tanisgazetesi.com.tr,hbrma.com,prokazan.ru,manisamans
 it-doc.info,v-androide.com,erkiss.live,igromania.ru,realnyezarabotki.com,healthhacks.ru,wd-x.ru,jianshu.com,memuplay.com,kino-teatr.ru,astrakhan-24.ru,rockstargame.su,hobbyits.com,eatcells.com,imgview.pw,abdullinru.ru,magazin.1001kadin.com,lingust.ru,1kinobig.ru,goblin-online.ru,it-tehnik.ru,netzpolitik.org,istasyongazetesi.com,kemergozcu.com,itop-gear.ru,golbasiguncel.com,haberamasya.com,suluovagazetesi.com,yesilirmakgazetesi.com,haberodasi.tv,amasyagazetesi.com,dunyabulteni.net,duzgunhaber.com.tr,geek.com,greasyfork.org,gunebakis.com.tr,hackworthy.blogspot.de,sarki-sozlerii.com,wunderground.com,zagrandok.ru###share
 !
 quoodo.com##.share_div
+belsat.eu##.tabs_container
```

Comment:
Why not .social_global ? Above name is too generic. Text of ext-CSS rule is not found in my code, but if ext-CSS rule is supposed to hide the header of .tabs_container, it is also hidden with .social_global.